src/components/album/AlbumGrid.js

```diff
@@ -14,4 +15,13 @@ function AlbumGrid({ albums, wishListId }) {
   );
 }
 
+AlbumGrid.propTypes = {
+  albums: PropTypes.arrayOf(
+    PropTypes.shape({
+      _id: PropTypes.string.isRequired,
+    })
```

Comment:
If you want to avoid having to retype long proptypes like the full album shape in multiple places, you can also centralize them.

You could have a folder `src/propTypes` with files for different prop-types eg: `albumTypes.js` and from there, do an `export const albumShape = PropTypes.shape(...)`. Then in both `Album.js` component and in `AlbumGrid.js` component you can import and reuse that prop type within the code (eg: `albums: PropTypes.arrayOf(albumtype)` )